rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1042,6 +1039,43 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	m_emu_settings->EnhanceComboBox(ui->vulkansched, emu_settings_type::VulkanAsyncSchedulerDriver);
 	SubscribeTooltip(ui->gb_vulkansched, tooltips.settings.vulkan_async_scheduler);
 
+	// Custom control that simplifies operation of two independent variables. Can probably be done better but this works.
+	ui->zcullPrecisionMode->addItems({"Precise (Default)", "Approximate (Fast)", "Relaxed (Fastest)"});
```

Comment:
This neither uses translation (preferably one per line), nor is it idiotproof by making sure that the data is passed on correctly (even if someone decides to sort it later for some reason).

So, add translation and pass the type as userData:

```suggestion
	ui->zcullPrecisionMode->addItem(tr("Precise (Default)"), static_cast<int>(emu_settings_type::PreciseZCULL));
	ui->zcullPrecisionMode->addItem(tr("Approximate (Fast)"), 0);
	ui->zcullPrecisionMode->addItem(tr("Relaxed (Fastest)"), static_cast<int>(emu_settings_type::RelaxedZCULL));
```

Reply:
See below.

Reply:
Added translation to the strings.